staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/config/client.go

```diff
@@ -54,8 +56,13 @@ func NewClientManager() (ClientManager, error) {
 	if err != nil {
 		return ClientManager{}, err
 	}
+	admissionScheme := runtime.NewScheme()
+	admissionv1beta1.AddToScheme(admissionScheme)
```

Comment:
This can return an error

Reply:
@ash2k Not necessary, because `admissionv1beta1.AddToScheme` will never return an error.

Reply:
If it will never return an error why does the function return an error? :) Tomorrow the implementation will change and something will not be right. E.g. some other code would be added before this function is called and it will cause it to error out. The worst thing about such situations is that it will be impossible to see the problem directly.
Russian proverb: "Even a stick shoots once a year" :)

Reply:
I agree that we should at least wrap those calls and trigger a panic. But this must be a pattern through the whole code base.